### PR TITLE
Add info block background to order page

### LINKED ISSUE #23115: Missing backgrounds on migrated order page
#### Describe the bug

Backgrounds/shadows of some elements are missing on order page.

#### Expected behavior

There should be grey backgrounds to separate the content and make it more readable.

**Screenshots**
bugs

#### Additional information

* PrestaShop version: develop
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Order page had info blocks with a gray background before, it should get back!
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | yes
| Deprecations?     | no
| Fixed ticket?     | Fixes #23115.
| How to test?      | Go on order page, and see if blocks get the grey background (see issue for more)
| Possible impacts? | Order page layout

# BC Break explanations
We removed a default var from the order file, and created a new more generic var inside variables files meaning that if a template inherit $order-block-background-color it won't exist anymore (but I doubt backoffice theme has been customized that much)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24283)

### CODE REVIEW

**Review by atomiix — APPROVED** [review]

File: admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss [hunk 1]
```diff
@@ -99,13 +99,12 @@
   margin-top: 8px;
 }
 
-$order-block-background-color: #f8f8f8 !default;
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss:102]: So if a theme inherit, it won't be able to use this variable ?

**Review by Progi1984 — COMMENTED** [review]
A small question

File: admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss:102]: yes right

Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss:102]: So could you put it back or define it as a BC Break ?

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
I'll wait for Progi's feedback to be addressed before sending the PR in QA 👍

**Comment by Robin-Fischer-PS** [comment]
Hello @NeOMakinG !

With your PR, I now have the background back OK (on grey, I used pink to highlight it).

Note that the not centered display of the total line in the Product block is another issue, not linked to this PR.

pr 24283 background

The rest of the page is displayed correctly too. I also tested in Mobile view, it's OK too.

So it's QA :heavy_check_mark: , thanks !

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @Robin-Fischer-PS

**Pull request merged**